pilz_psenscan_test_area/execute_test.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash -x
+
+source_ws()
+{
+  source "$CATKIN_WS_DIR/devel/setup.bash" 2>&1
+}
```

Comment:
I would move this to the `automatic_acceptance_test_psen_scan.sh` script or the `setup_workspace.sh` script.

Reply:
done